br/tests/docker_compatible_gcs/_run.sh

```
@@ -40,7 +42,7 @@ export GOOGLE_APPLICATION_CREDENTIALS="br/tests/$TEST_NAME/config.json"
 for TAG in ${TAGS}; do
     echo "restore ${TAG} data starts..."
     # after BR merged into TiDB we need skip version check because the build from tidb is not a release version.
-    bin/br restore db --db test -s "gcs://$BUCKET/bk${TAG}" --pd $PD_ADDR --gcs.endpoint="http://$GCS_HOST:$GCS_PORT/storage/v1/" --check-requirements=false
+    bin/br restore db --db test -s "gcs://$BUCKET/bk${TAG}" --pd $PD_ADDR --gcs.endpoint="http://$GCS_HOST:$GCS_PORT" --check-requirements=false
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)